example/ios/Podfile

```
@@ -0,0 +1,35 @@
+  use_react_native!(
+    :path => config[:reactNativePath],
+    # An absolute path to your application root.
+    :app_path => "#{Pod::Config.instance.installation_root}/.."
+  )
+
+  post_install do |installer|
+    # https://github.com/facebook/react-native/blob/main/packages/react-native/scripts/react_native_pods.rb#L197-L202
+    react_native_post_install(
+      installer,
+      config[:reactNativePath],
+      :mac_catalyst_enabled => false,
```

RuboCop (1.69.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)